test/docker-compose.db.yml

```diff
@@ -0,0 +1,26 @@
+version: "2"
+services:
+
+  test:
+    build:
+      context: ..
+      dockerfile: ./test/Dockerfile.py3
```

Comment:
currently only testing on py3. Not sure if we should also include py2. It's probably not worth it since we don't use this db access in AB at the moment.